spec/feature/users_controllers_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe 'Users', type: :feature do
+  before(:all) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'Teacher from Mexico.', post_counter: 1)
+  end
+  it 'should have a list of users' do
+    visit '/users'
+    expect(page).to have_content('Tom')
+    expect(page).to have_content('Number of posts: 1')
+  end
```

Comment:
- [ ] Kindly make sure the `show` action for `User` is also tested as required for the project. Right now you only tested the `index` action. I suggest you do it like in the example below to make your test clearer about `index` and `show` actions.
e.g.: 
```ruby
  describe 'GET /index' do
    ...
  end

  describe 'GET /show' do
    ...
  end
```

- [ ] Kindly make sure you also include tests that ensure that the correct view template is rendered as required for the project.
Example : `expect(response).to render_template(:show)` or `expect(response).to have_http_status(:ok)`,...

Reply:
Done